### PR TITLE
Completed Responsive Web Design

### PR DESCRIPTION
**Pull request by appiersign**
#### In This Pull Request We Worked on:
- adding semantic HTML elements
- styling various element to match the original website
- used both flexbox and the grid template to make the site responsive
- added font awesome icons for social media links
- added a gitignore file to exempt files and directories from being tracked
- fixed html validated errors

### CODE REVIEW

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi!

The project is looking great 🥇, very nice use of Flexbox and CSS Grid, also a nice use of semantic HTML.

There are some requirements that are not met yet:

### Small version
* [x] Navigation and sub-navigation row change to a hamburger (bars) icon at the rightmost part (next to search icon).
### Medium version
* [x] Top 3 articles the TWN logo at the top left corner of the row disappears:
image
### Large version
* [x] Lastest deal section aligns 4 deals per row, this is how it is looking now:
image

Status: **Request changes**.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

**Hi Team**, congratulations on the effort in this project :+1:   Your project looks really great but for the project to be approved, we have to make small changes to the project :slightly_smiling_face: Below is the comment:
- It contains social media icons
smi
Like this
like

- Kindly check the horizontal scroll at a small screen size
SC
- It contains TNW logo
TNW
Like this
like2

Kindly make these changes and submit code review :100: 
Happy coding :computer: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo-9738b0128/)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,671 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Responsive Design</title>
+  <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
+  <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> -->
```
Comment (pelzolga123) [inline index.html]: This commented line should be removed.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: great!
Here are the things that need fixing:
* [x] First section is a little messed up in large and medium versions, you need to align pictures properly. Right now it looks like this:
image
* [x] It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
 
### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
## Hi team!
### Nice job! Project looks! :+1: 

## Status [Approved]







#### Go ahead and merge to master.
#### Goodluck in your next project Happy coding

**Review by Anna-Myzukina — APPROVED** [review]

**Pull request merged**